charts/gardener/controlplane/charts/application/templates/rbac-user.yaml

```diff
@@ -16,28 +16,56 @@ rules:
   resources:
   - '*'
   verbs:
-  - '*'
+  - create
```

Comment:
Quick question. Shouldn't `manage-members` and `modify-spec-tolerations-whitelist` verbs for Projects, and  `modify-spec-kubernetes`, `modify-spec-machineimages`, `modify-spec-providerconfig` verbs for NamespacedCloudProfiles also be explicitly specified here?